src/broadcom/common/v3d_debug.h

```diff
@@ -57,7 +57,7 @@ extern uint32_t V3D_DEBUG;
 
 #ifdef HAVE_ANDROID_PLATFORM
 #define LOG_TAG "BROADCOM-MESA"
-#include <cutils/log.h>
+#include <log/log.h>
```

Comment:
As tapani proposed let's have support for both old and new header includes ? we can than send this to public mailing list ?

Reply:
@tpalli how to support both old and new header includes? adding a MARCO ifdef for android? on Android support new log header includes while on other platform support old header includes.
Bug this header includes is only effect on android plateform since it is in a MARCO ifdef HAVE_ANDROID_PLATFORM
what is the preferred way in mesa to support both header includes?